.github/workflows/CI.yml

```diff
@@ -124,14 +124,14 @@ jobs:
         env:
           OS_MIGRATIONS_GRADLE_SCAN_TOS_AGREE_AND_ENABLED: ''
       - name: Install python dependencies
-        working-directory: test
+        working-directory: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test
         run: |
           python -m pip install --upgrade pipenv
           pipenv install --deploy --dev
```

Comment:
This can be deleted, python is running on the docker instance, not on the github action runner

Reply:
In fact, I think we can remove all the python logic from this part of the workflow.

Reply:
Yes have removed this now